web/src/main/java/com/graphhopper/http/GraphHopperWeb.java

```diff
@@ -152,7 +154,9 @@ public static PathWrapper createPathWrapper(JSONObject path,
 
                     // The translation is done from the routing service so just use the provided string
                     // instead of creating a combination with sign and name etc
-                    instr.setUseRawName();
+                    // However, you can translate it on the client side by passing translate_insructions=false
```

Comment:
translate_instructions=false is the default. So I thought it would be better to describe the "special case". Should I change it?